rust/exchange_rate/README.md

```diff
@@ -57,7 +57,9 @@ triggering remote HTTP calls again.
 `cargo build --target wasm32-unknown-unknown --release --p exchange_rate`
 
 ## Deploy the canister locally
+- `cargo install cargo-audit` to install vulnerability inspection tool
+- `npm install` to prepare needed UI canister dependencies
 - `dfx start --enable-canister-http` to start a local IC instance with Canister HTTP feature enabled
-- `dfx deploy --with-cycles 100000000000` to deploy the `exchange_rate` and `exchange_rate_assets`
+- `dfx deploy --with-cycles=40000000000000` to deploy the `exchange_rate` and `exchange_rate_assets`
```

Comment:
Motoko version currently has no README at all.

Reply:
Will create one.

Reply:
Have created `motoko` README and checked in.